docs/examples/DataSet/Working-With-Pandas-and-XArray.ipynb

```diff
@@ -185,14 +195,77 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 7,
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
+     "name": "stderr",
      "output_type": "stream",
      "text": [
-      "Starting experimental run with id: 125\n"
+      "2020-04-15 10:25:37,830 ¦ qcodes.dataset.sqlite.connection ¦ ERROR ¦ connection ¦ atomic ¦ 106 ¦ Rolling back due to unhandled exception\n",
```

Comment:
Could you rerun this one ?

Reply:
Thanks for catching this. It happened most likely because I was running all notebooks at once.